docs/make_bulletin_json.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+# Set the file paths
+input_md_file = os.path.join(os.path.dirname(__file__), "bulletin.md")
+output_json_file = os.path.join(os.path.dirname(__file__), "bulletin.json")
+
+
+def generate_json_file():
+    with open(input_md_file, "r", encoding="utf-8") as md_file:
+        markdown_content = md_file.read()
+    bulletin_json = []
+    content = ""
+
+    # Initialize title and date with default values
+    title = "DEFAULT_TITLE"
+    date = "DEFAULT_DATE"
+
+    for line in markdown_content.split("\n"):
+        # Skip if the line begins with #
+        if line.startswith("# "):
+            continue
+        elif line.startswith("---"):
+            bulletin_json.append({"title": title, "date": date, "content": content})
+            content = ""
+            # Reset title and date to their default values after each section
+            title = "DEFAULT_TITLE"
+            date = "DEFAULT_DATE"
+        elif line.startswith("## "):
+            title = line[3:].strip()
+        elif line.startswith("_"):
+            date = line[1 : len(line) - 1].strip()
```

Comment:
This probably needs a stronger check, like a date format regexp or another more explicit indicator. There are lots of cases where we might start a line with `"_"` in the main content.

Reply:
I have added a regex that checks if the entire line has only the date content and it is in a certain format.